### PR TITLE
Lesson6

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

File: lesson6/public/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <div>
+      <pre id = 'm1'/>
+ </div> 
+ <script>
+const button = document.getElementById('subBox');
+button.onclick=()=>{
+  const name = document.getElementById('inputBox').value;
+  const comment = document.getElementById('comBox').value;
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('GET', `/XMLHttpRequest?nameVal=${name}&comVal=${comment}`);
```
Comment (MadhuBeesetty) [inline lesson6/public/index.html]: /XMLHttpRequest can be a bit simpler , so that you wont get confused.

Comment (saumya665) [reply lesson6/public/index.html]: Thanks Madhu! will change the name of the request

**Review by ameyunaikgithub — COMMENTED** [review]
hey is this indented?

File: lesson5/public/index.html [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-    <title></title>
-  </head>
-  <body>
-      Name: <input type = 'text' id = 'nameBox'/>
-      Comment: <input type = 'text' id = 'commentBox'/>
-      <input type = 'submit' id = 'submitBox'>
-    <script>
-        const button = document.getElementById('submitBox');
-        button.onclick =()=>{
-          const name = document.getElementById('nameBox').value;
```
Comment (ameyunaikgithub) [inline lesson5/public/index.html:14]: is it indented, check it once again

Comment (saumya665) [reply lesson5/public/index.html:14]: this file is part of lesson5 so I removed it. Please review the files under lesson6.

File: lesson6/public/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:   <input type = 'text' id = 'inputBox'/>
+    Comment: <input type = 'text' id = 'comBox'/>
+    <input type = 'submit' id = 'subBox'/>
+    <div>
+      <pre id = 'm1'/>
```
Comment (seemcat) [inline lesson6/public/index.html]: Provide better id names. This can be called messages.

Comment (seemcat) [reply lesson6/public/index.html]: You can start implementing it on the next lesson.

**Review by seemcat — APPROVED** [review]

File: lesson6/public/index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('GET', `/XMLReq?nameVal=${name}&comVal=${comment}`);
+  xhttp.send();
+}
+const getChatFile =()=>{
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('GET', '/textOut.txt');
+  xhttp.onreadystatechange=()=>{
+    if(xhttp.responseText){
+      const content = document.getElementById('m1');
```
Comment (MadhuBeesetty) [inline lesson6/public/index.html]: try to give better variable names

**Review by MadhuBeesetty — APPROVED** [review]

**Pull request merged**